iree/compiler/Dialect/HAL/Target/WebGPU/SPIRVToWGSL.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright 2021 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Dialect/HAL/Target/WebGPU/SPIRVToWGSL.h"
+
+#include "llvm/Support/Debug.h"
+#include "llvm/Support/raw_ostream.h"
+#include "tint/tint.h"
```

Comment:
I like that this is sequestered in here - not having tint.h (and its transitive deps) leak into our other files insulates us from all kinds of include order/etc issues.